### PR TITLE
1806 manufacture

### LINKED ISSUE #1806: Implement MARC `264$#3$` (Manufacture)
[HT016692738](https://lobid.org/resources/990183958380206441) should include the "Manufacture place and date" ([MARC 264$#3$](https://www.loc.gov/marc/bibliographic/bd264.html)); possibly modelled after `264$#1$` (`publication`).

```xml
<datafield tag="264" ind1=" " ind2="3">
  <subfield code="a">Marsberg</subfield>
  <subfield code="b">Boxberger</subfield>
  <subfield code="c">2010</subfield>
</datafield>
```

Also: Would it make sense to use [008/07-10](https://www.loc.gov/marc/bibliographic/bd008a.html) (Date 1) as fallback for `publication.startDate`?

Originally reported as DigiBib-Ticket#2023062250336954.

### PR DESCRIPTION
**Pull request by TobiasNx**
Resolves #1806

Add manufacture analog to publication.

Needs to be done/answered, @acka47 can you help:
- Decide if we just add manufacture as lobid vocab or find a fitting property @acka47 
- Add property and context for manufacture
- Decide about the type. If [`sdo:Event`](https://schema.org/event)  is good enough, since it is no PublishingEvent and Event is the "parent"(?) class.

@blackwinter is this what you had in mind?

### CODE REVIEW

**Review by blackwinter — APPROVED** [review]
Yes.

File: src/test/resources/alma-fix/990185619180206441.json [hunk 1]
```diff
@@ -12,6 +12,11 @@
     "location" : [ "s.l." ],
     "publishedBy" : [ "EMI Records" ]
   } ],
+  "manufacture" : [ {
+    "startDate" : "1994",
+    "type" : [ "Event" ],
+    "location" : [ "EMI Records Ltd, p 1969" ]
```
Comment (acka47) [inline src/test/resources/alma-fix/990185619180206441.json:18]: This seems wrong. It should rather read "manufacturer" or similar and we should put the actual place of manufacture in this field.

**Review by acka47 — CHANGES_REQUESTED** [review]
As commented

File: src/test/resources/alma-fix/990185619180206441.json (hunk 1 above)
Comment (TobiasNx) [reply src/test/resources/alma-fix/990185619180206441.json:18]: This is written in:
`$a - Place of production, publication, distribution, manufacture (R)`

this seems to be a cataloguing error.

File: src/test/resources/alma-fix/990183958380206441.json [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+  "deprecatedUri" : "http://lobid.org/resources/HT016692738#!",
+  "isbn" : [ "3980031357", "9783980031356" ],
+  "oclcNumber" : [ "706989442" ],
+  "title" : "Rösenbeck",
+  "otherTitleInformation" : [ "Orts- und Landesgeschichte am Beispiel eines Bredelarer Klosterdorfes" ],
+  "manufacture" : [ {
+    "startDate" : "2010",
+    "type" : [ "Event" ],
+    "location" : [ "Marsberg" ],
+    "manufacturedBy" : [ "Boxberger" ]
```
Comment (TobiasNx) [inline src/test/resources/alma-fix/990183958380206441.json:14]: @acka47 This is a better example.

**Comment by TobiasNx** [comment]
@dr0i you can merge this

**Comment by dr0i** [comment]
Will be deployed next Monday.

**Comment by dr0i** [comment]
The `manufacture` and `manufacturedBy` properties are missing in the context(<= labels.json).

**Pull request merged**